prow/images/kyma-integration/Dockerfile

```diff
@@ -163,3 +163,13 @@ RUN wget -q https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cl
 #################################################################
 
 RUN go get golang.org/x/perf/cmd/...
+
+#################################################################
+####################### Install smctl ###########################
+#################################################################
+
+ENV SMCTL_VERSION=v1.11.10
+RUN wget -q https://github.com/Peripli/service-manager-cli/releases/download/${SMCTL_VERSION}/smctl-v1.11.10-linux-amd64.tar.gz && \
```

Comment:
the `smctl-v1.11.10` string can be replaced with the SMCTL_VERSION variable

Reply:
thanks for spotting this, done